klasses/management/commands/manage_certificates.py

```diff
@@ -0,0 +1,130 @@
+        group.add_argument(
+            "--generate",
+            help="Flag for single certificate generation",
+            dest="generate",
+            action="store_true",
+            required=False,
+        )
+        group.add_argument(
+            "--generate-batch",
+            help="Flag for batch certificate generation",
+            dest="generate_batch",
+            action="store_true",
+            required=False,
+        )
+
+        super().add_arguments(parser)
+
+    def handle(self, *args, **options):  # pylint: disable=too-many-locals
+        """Handle command execution"""
+        try:
+            user = fetch_user(options["user"]) if options["user"] else None
+            generate_single = options.get("generate")
+            generate_batch = options.get("generate_batch")
+            bootcamp_run = fetch_bootcamp_run(options.get("run"))
+            revoke = options.get("revoke")
+            unrevoke = options.get("unrevoke")
+        except:
+            raise CommandError("Provided values are not valid.")
+
+        if generate_single and (not user or not bootcamp_run):
+            raise CommandError(
+                "A valid 'user' and 'run' must be provided with 'generate'."
+            )
+        if generate_batch and not bootcamp_run:
+            raise CommandError("A valid 'run' must be provided with 'generate-batch'.")
+        if (revoke or unrevoke) and (not user or not bootcamp_run):
+            raise CommandError(
+                "A valid 'user' and 'run' must be provided with 'revoke' or 'unrevoke'"
+            )
+
+        if revoke and user and bootcamp_run:
+            updated = revoke_certificate(user, bootcamp_run)
+            self.show_message(
+                updated,
+                message="Certificate has been revoked for user:{} and bootcamp-run:{}".format(
+                    user.email, bootcamp_run
+                ),
+            )
+        elif unrevoke and user and bootcamp_run:
+            updated = unrevoke_certificate(user, bootcamp_run)
+            self.show_message(
+                updated,
+                message="Certificate has been un-revoked for user:{} and bootcamp-run:{}".format(
+                    user.email, bootcamp_run
+                ),
+            )
+        elif generate_single and bootcamp_run and user:
+            updated = generate_single_certificate(user, bootcamp_run)
+            self.show_message(
+                updated,
+                message="Certificate has been created for user:{} and bootcamp-run:{}".format(
+                    user.email, bootcamp_run
+                ),
+            )
+        elif generate_batch and bootcamp_run:
+            updated = generate_batch_certificates(bootcamp_run)
+            self.show_message(
+                updated,
+                message="{} new certificates have been created for bootcamp-run:{}".format(
+                    len(updated), bootcamp_run
+                ),
+            )
```

Comment:
Can we get to this point and none of the conditions have been met? Should we have a generic error?